packages/console/src/components/TenantEnvTag/index.tsx

```diff
@@ -9,24 +9,34 @@ import * as styles from './index.module.scss';
 type Props = {
   tag: TenantTag;
   className?: string;
+  isFullName?: boolean;
```

Comment:
nit: This is a bit confusing as I didn't get it when I first saw it. Maybe some jsdoc would help. Or you can probably rename it to `isAbbreviated`

Reply:
Updated.